spec/controllers/answers_controller_spec.rb

```diff
@@ -1,15 +1,39 @@
 require 'rails_helper'
+require 'pp'
 
 describe AnswersController do
-  describe 'GET routes' do
-    it "GET questions/x/answers"
-    it "GET questions/x/answers/y"
-  end
+  let(:user){User.create(username: "Leo Dongchul", password: '123', email: "therio@gmail.com")}
+  let(:question) {Question.create(title: "Who is DongChul ?", content: "He is mysterious and mindful, who is he, really ?", category_id: 1, user: user)}
+
+  context 'hey' do
+    before(:each) do
+      user
+      question
```

Comment:
You should not have to do this.  If you _do_, there are better ways to force early creation.  See `let!` versus `let` in RSpec.